app/controllers/requests_controller.rb

```diff
@@ -40,6 +40,20 @@ def start
     redirect_to new_distribution_path(request_id: request.id)
   end
 
+  def print_unfulfilled
+    requests = Request.includes(partner: [:profile]).where(status: [0, 1])
```

Comment:
Wait - this isn't scoped by organization, no? This will pull every request from every organization.

Reply:
This should also include `item_requests` - we should be using those instead of `request_items`, which is a leftover from the days when we had two databases.

Reply:
Also, can you use `status: [Request.statuses[:pending], Request.statuses[:started]]`? That's more obvious than 0 and 1.

Reply:
@dorner I'm working to address your 1st and 3rd comment. I have a question about the 2nd one:

> This should also include item_requests - we should be using those instead of request_items, which is a leftover from the days when we had two databases.

I recall talking about `item_requests` and `request_items` in office hours and learning that one was the chosen way going forward but I had it the other way around that `request_items` was what we were keeping. Since I lack the historical context for this, I want to double check before making changes.

Reply:
It's `item_requests` that's the real one. That's a fully fledged relational model, as opposed to a serialized JSON object which is harder to work with.